### ISSUE TITLE
Allow wrapping of messages

### ISSUE THREAD

**Issue by aaronjensen** [wallabyjs/atom-wallaby#9]
![booking-site - atom 2015-09-09 17-08-27](https://cloud.githubusercontent.com/assets/8588/9777157/68cc41f4-5715-11e5-989b-2a7f01f4beff.png)
Right now, if an inline message appears and gets cut off by the edge of the window or pane, it doesn't appear that there is any way to see it other than to open the bottom panel. Maybe i'm missing something?

**Comment by ArtemGovorov** [comment]
Yes, it's the case in all of the editors that we support. Besides a few technical difficulties in core, there're a few of other concerns that needs to be addressed: 
- we still need to truncate the message (and we do), so to see the full message one would need another way anyway;
- spreading the message over a few lines may be confusing for a user, it may also clash with other console messages/error messages.

There's a way that perhaps may be a bit more attractive for you than using the bottom panel - it's using wallaby `Show line tests` command:

![st](https://cloud.githubusercontent.com/assets/979966/9777552/32955f64-57a9-11e5-9199-d0a5cf9281df.gif)

One other trick that I occasionally use is this:
<img width="751" alt="screen shot 2015-09-10 at 10 56 11 am" src="https://cloud.githubusercontent.com/assets/979966/9777668/9e39c146-57aa-11e5-8eaa-152436b215fc.png">

**Comment by ummahusla** [comment]
@ArtemGovorov Well, as I already mentioned that I haven't found the way of scrolling to the right in the output. 

![image 2017-01-03 at 3 14 22 pm](https://cloud.githubusercontent.com/assets/2536096/21634932/7ed17122-d252-11e6-8188-d3ef8691fcd7.png)

Hopefully, at the moment, we still can check the error log and the bottom to have a better understanding where do the errors are coming from. 

![1fd0dcda773953d5e13d960a68178c02 _image 202017-01-04 20at 207 48 38 20am](https://cloud.githubusercontent.com/assets/2536096/21634966/be0adb6c-d252-11e6-9284-16c07eb3d69a.png)

Personally, I still think that we need to think how we can handle that scrolling directly in the editor body.
